### ISSUE TITLE
Does not accept service uuids to filter in parameters for startScanning() like other biindings?

### ISSUE THREAD

**Issue by pursual** [Timeular/noble-winrt#18]
The noble lib allows you to provide two parameters to startScanning() (an array of service uuids, and a boolean for allow duplicates).

This binding does not discover anything when an array of service ids is passed?

**Comment by pursual** [comment]
It looks like it works when the array only contains one item?

**Comment by geovie** [comment]
@pursual You're right currently we pass the service uuids as [filter](https://github.com/Timeular/noble-winrt/blob/master/src/ble_manager.cc#L124) to the AdvertismentWatcher.

The alternative would be to watch for all and filter the advertisement manually once we receive it.

If anyone wants to provide a PR for this the steps would be:
- remove the advertisment filter https://github.com/Timeular/noble-winrt/blob/master/src/ble_manager.cc#L119-L127
- save `serviceUUIDs` in a class field `mAdvertiementServiceUUIDs`  (similar to `mAllowDuplicates`)
- check if the received advertisement's [serviceUUIDs](https://docs.microsoft.com/en-us/uwp/api/windows.devices.bluetooth.advertisement.bluetoothleadvertisement.serviceuuids?view=winrt-19041#Windows_Devices_Bluetooth_Advertisement_BluetoothLEAdvertisement_ServiceUuids) contain *any* of the saved `mAdvertiementServiceUUIDs` https://github.com/Timeular/noble-winrt/blob/master/src/ble_manager.cc#L139

**Comment by tony-gutierrez** [comment]
Fixed in https://github.com/tony-gutierrez/noble-winrt/tree/servicefilter which is based off of https://github.com/fido-alliance/noble-winrt
